src/contact/components/EditInfo.js

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+
+export default class EditInfo extends React.Component {
+	constructor(props){
+		super(props);
+	}
+	componentDidMount(){
+		if(this.props.info === null){
+			this.props.actions.displayInfo(true);
+		} else {
+			this.props.actions.displayInfo(true,this.props.info);
+		}
+	}
+
+	addEmailField(){
+		this.props.actions.addEmail();
+	}
+
+	addPhoneField(){
+		this.props.actions.addTel();
+	}
+	
+	onInputChange = (e,field) => {
```

Comment:
Why execute this only on onChange?

Reply:
This is for error cleaning if required field has value